<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraControlDialog</class>
 <widget class="QDialog" name="CameraControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UVC Camera Control</string>
  </property>
  <widget class="QTabWidget" name="tabWidgetControl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>540</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>QCamera</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonDetectCamera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>250</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>FindCamera</string>
     </property>
    </widget>
    <widget class="QLabel" name="labellResolution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxFormats">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>200</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboVideoDevices">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>200</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelCamera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>265</width>
       <height>405</height>
      </rect>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::TabShape::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSettings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <property name="spacing">
           <number>9</number>
          </property>
          <item row="10" column="1">
           <widget class="QComboBox" name="dropdownWhiteBalanceMode">
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Manual</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sunlight</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cloudy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shade</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tungsten</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fluorescent</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Flash</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sunset</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QPushButton" name="buttonFFMPEG">
            <property name="text">
             <string>Open FFMPEG Settings</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelAutoExposureTime">
            <property name="text">
             <string>Auto Exposure </string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelManualIsoSensitivity">
            <property name="text">
             <string>  ISO Sensitivity</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelAutoIsoSensitivity">
            <property name="text">
             <string>Auto Iso Sensitivity</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSlider" name="sliderManualIsoSensitivity">
            <property name="maximum">
             <number>128000</number>
            </property>
            <property name="pageStep">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="dropdownFlashMode">
            <item>
             <property name="text">
              <string>Off</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>On</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelFlashMode">
            <property name="text">
             <string>Flash Mode</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="sliderManualExposureTime">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="checkboxAutoExposureTime">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QCheckBox" name="checkboxAutoIsoSensitivity">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QComboBox" name="dropdownTorchMode">
            <item>
             <property name="text">
              <string>Off</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>On</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="dropdownExposureMode">
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Manual</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Portrait</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Night</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sports</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Snow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Beach</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Action</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Landscape</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Night Portrait</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Theatre</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sunset</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Steady Photo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fireworks</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Party</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Candlelight</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Barcode</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="labelWhiteBalanceMode">
            <property name="text">
             <string>White Balance Mode</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="sliderExposureComp">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelManualExposureTime">
            <property name="text">
             <string>   ExposureTime</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="labelTorchMode">
            <property name="text">
             <string>Torch Mode</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelExposureMode">
            <property name="text">
             <string>Exposure Mode</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="sliderColorTemp">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="pageStep">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelExposureComp">
            <property name="text">
             <string>Exposure Comp.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelColorTemperature">
            <property name="text">
             <string>Color Temperature</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabZoomFocus">
      <attribute name="title">
       <string>Zoom and Focus</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="title">
          <string>Zoom and Focus</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="labelFocusDistance">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Focus Distance</string>
            </property>
            <property name="buddy">
             <cstring>sliderFocusDistance</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelZoomRate">
            <property name="text">
             <string>Zoom Rate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelZoomFactor">
            <property name="text">
             <string>Zoom Factor</string>
            </property>
            <property name="buddy">
             <cstring>sliderZoom</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelFocusMode">
            <property name="text">
             <string>Focus Mode</string>
            </property>
            <property name="buddy">
             <cstring>dropdownFocusMode</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="sliderFocusDistance">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="dropdownFocusMode">
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Auto (Near)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Auto (Far)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hyperfocal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Infinity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Manual</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="sliderZoom">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="sliderZoomRate">
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>OpenCV</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonInitOpenCVCapture">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>250</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Init VideoCapture</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCapture">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VideoCapture Index</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboVideoCapture">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxCaptureSetting">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>VideoCapture Setting</string>
     </property>
     <widget class="QLabel" name="labelWidth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditWidth">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditHeight">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeight">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFps">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>40</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelFps">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>70</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Format / Fps</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFormat">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonGetParam">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>90</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Get property</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonControl">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera control</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>480</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>250</width>
       <height>280</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::LineWrapMode::WidgetWidth</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditFormat</tabstop>
  <tabstop>lineEditFps</tabstop>
  <tabstop>lineEditWidth</tabstop>
  <tabstop>lineEditHeight</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>dropdownWhiteBalanceMode</tabstop>
  <tabstop>buttonFFMPEG</tabstop>
  <tabstop>sliderManualIsoSensitivity</tabstop>
  <tabstop>dropdownFlashMode</tabstop>
  <tabstop>sliderManualExposureTime</tabstop>
  <tabstop>checkboxAutoExposureTime</tabstop>
  <tabstop>checkboxAutoIsoSensitivity</tabstop>
  <tabstop>dropdownTorchMode</tabstop>
  <tabstop>dropdownExposureMode</tabstop>
  <tabstop>sliderExposureComp</tabstop>
  <tabstop>sliderColorTemp</tabstop>
  <tabstop>sliderFocusDistance</tabstop>
  <tabstop>dropdownFocusMode</tabstop>
  <tabstop>sliderZoom</tabstop>
  <tabstop>sliderZoomRate</tabstop>
  <tabstop>pushButtonInitOpenCVCapture</tabstop>
  <tabstop>comboVideoCapture</tabstop>
  <tabstop>comboBoxFormats</tabstop>
  <tabstop>comboVideoDevices</tabstop>
  <tabstop>tabWidgetControl</tabstop>
  <tabstop>pushButtonDetectCamera</tabstop>
  <tabstop>pushButtonGetParam</tabstop>
  <tabstop>pushButtonControl</tabstop>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
